### PR TITLE
Differentiate between `push` and `pull` `mirror sync in progress`

### PR DESCRIPTION
**Pull request by delvh**
Previously, if you had both a push and a pull mirror, the message did not clarify if you've accidentally synchronized the wrong one.
Additionally fixed two typos that were encountered while debugging.

## Screenshots
grafik
grafik

### CODE REVIEW

**Comment by delvh** [comment]
Note: I'm not too happy about the text wording but couldn't think of anything better.
If anyone has suggestions, let me know.

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -2009,7 +2009,8 @@ settings.mirror_settings.push_mirror.add = Add Push Mirror
 settings.mirror_settings.push_mirror.edit_sync_time = Edit mirror sync interval
 
 settings.sync_mirror = Synchronize Now
-settings.mirror_sync_in_progress = Mirror synchronization is in progress. Check back in a minute.
+settings.pull_mirror_sync_in_progress = Syncing changes from the remote %s is currently in progress.
```
Comment (delvh) [inline options/locale/locale_en-US.ini]: Alternatively,
```suggestion
settings.pull_mirror_sync_in_progress = Pulling changes from the pull mirror %s at the moment.
```
or
```suggestion
settings.pull_mirror_sync_in_progress = Pulling changes from the origin %s at the moment.
```

Comment (lunny) [reply options/locale/locale_en-US.ini]: Pulling changes from the remote %s at the moment.

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -2009,7 +2009,8 @@ settings.mirror_settings.push_mirror.add = Add Push Mirror
...
+settings.push_mirror_sync_in_progress = Pushing changes to the push mirror %s at the moment.
```
Comment (lunny) [inline options/locale/locale_en-US.ini]: Pushing changes to the remote %s at the moment.

**Review by lunny — APPROVED** [review]

**Review by puni9869 — APPROVED** [review]

**Pull request merged**